### DISCUSSION TITLE
Type: Internal/External

### DISCUSSION THREAD

**Discussion by Tealk** [goauthentik/authentik#7520] [Q&A]
What does the type internal or external mean?

I have a public authentication instance and only one registration path but different details for the users.

**Comment by ser** [comment]
From what I understan it means simply nothing at this stage :-) They plan probably some extra features in future for paying customers and you will have to pay for "internal" more than for "external" :)

**Reply by Tealk** [reply]
ok, but then it shouldn't be randomly selected  xD
